### PR TITLE
Contributing instructions

### PR DESCRIPTION
**Pull request by romainmartinez**
This PR add the [contributing page](https://pyomeca.github.io/contributing/).

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Keep any changes in this branch specific to one bug or feature so it is clear what the branch brings to pyomeca.
+
+## Testing your code
+
+Adding tests is required if you add or modify existing codes in pyomeca.
+Therefore, it is worth getting in the habit of writing tests ahead of time so this is never an issue.
+The pyomeca test suite runs automatically on GitHub Actions, once your pull request is submitted.
+However, we strongly encourage running the tests prior to submitting the pull request.
+To do so, simply run `make test`.
```
Comment (pariterre) [inline docs/contributing.md:48]: This one is problematic though, since here, from the PyCharm debugger, I could not run the tests...

Comment (romainmartinez) [reply docs/contributing.md]: I added instruction to run tests without `make` in https://github.com/pyomeca/pyomeca/pull/97/commits/2e22b5a5518daaa9da06ef3ca6d3ccdd3de2d0ea.
However, in your case, it boils down to an editor configuration (see [here](https://www.jetbrains.com/help/pycharm/pytest.html)). Given the multiple editors, I'm not sure I want to give explanations for all of them. The user is free to use `pytest` — which is fairly standard — as he sees fit.

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+You will need your own fork to work on the code.
+Go to the [pyomeca project page](https://github.com/pyomeca/pyomeca/) and hit the `Fork` button.
+You will want to clone your fork to your machine:
+
+```bash
+git clone https://github.com/your-user-name/pyomeca.git
+cd pyomeca
+git remote add upstream https://github.com/pyomeca/pyomeca.git
+```
```
[the remark this answers is not in the document]
Comment (romainmartinez) [reply docs/contributing.md:15]: While I agree that I have not tested all commands on Windows (in fact none of them :rofl: ), all of them should work on all OS.
`git` and `make` are both available on windows, mac and linux.
I completely agree that this section is more "low level" that the rest of the documentation but the target audience here is other developers.

**Comment by romainmartinez** [comment]
This PR is ready to merge if you are OK

**Pull request merged**